src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1454,6 +1454,10 @@ private BoundExpression GetDefaultParameterValue(SyntaxNode syntax, ParameterSym
                 defaultValue = MakeConversionNode(defaultValue, parameterType, @checked: false, acceptFailingConversion: true);
             }
 
+            // CONSIDER: it feels like determining the exact value that will be emitted for a
+            // parameter default value is something that should be figured out at binding time, not in lowering.
+            defaultValue = defaultValue.WithSuppression(syntax.IsKind(SyntaxKind.SuppressNullableWarningExpression));
```

Comment:
>defaultValue [](start = 12, length = 12)

Just curious, did you understand why that bound node doesn't carry the proper suppression flag already? That's rather surprising since it's handled by `BindSuppressNullableWarningExpression`

Reply:
Yeah, I think we could work around this here by somehow obtaining the bound expression for the default value, but I do not think it's worth it for now. Better to do the work in the future to factor this in a more sane way. #47352